cogs/challenges.py

```diff
@@ -21,16 +33,86 @@ async def challenges_group(self, ctx: commands.Context) -> None:
         aliases=("rs",),
         brief="Resubmit Command to remove submitted role"
     )
-    @commands.has_any_role(STAFF_ROLE_ID, CHALLENGE_HOST_HELPER_ROLE_ID) # Staff role or challenge host helper
     async def challenges_resubmit(self, ctx: commands.Context, member: discord.Member):
-        
-        submitted_role = ctx.guild.get_role(SUBMITTED_ROLE_ID)  # Submitted role
+        submitted_role = ctx.guild.get_role(CHALLENGE_SUBMITTED_ROLE_ID)  # Submitted role
 
         if submitted_role in member.roles:  # Checking is user has the submitted role
             await member.remove_roles(submitted_role)
             return await ctx.send(f"Submitted role removed from {member.mention}")
     
-        return await ctx.send(f"Member doesn't have the submitted role")
+        return await ctx.send("Member doesn't have the submitted role")
+
+    @challenges_group.command(
+        name="announce_winners",
+        aliases=("aw",),
+        brief=("Command to announce the distribution of :pancakes:")
+    )
+    async def announce_winners(self, ctx: commands.Context):
+        info_channel = ctx.guild.get_channel(CHALLENGE_INFO_CHANNEL_ID)
+
+
+        return await info_channel.send(
+            f"<@&{CHALLENGE_WINNER_ROLE_ID}> :pancakes: have been given out, " \
+            f"go deposit them in <#{BOT_GAMES_CHANNEL_ID}>. \n" \
+            f"Analysis for the challenge will be available shorly in <#{CHALLENGE_INFO_CHANNEL_ID}>",
+            allowed_mentions=discord.AllowedMentions(
+                roles=[
+                    discord.Object(CHALLENGE_WINNER_ROLE_ID)
+                ]
+            )
+        )
+
+    @challenges_group.command(
+        name="open_submissions",
+        aliases=("os",),
+        brief=("Command to open submissions")
+    )
+    async def open_submissions(self, ctx: commands.Context):
+        info_channel = ctx.guild.get_channel(CHALLENGE_INFO_CHANNEL_ID)
+
+        submit_channel = ctx.guild.get_channel(CHALLENGE_SUBMIT_CHANNEL_ID)
+        
+        # Allows people to submit
+        overwrite = submit_channel.overwrites_for(ctx.guild.default_role)
+        overwrite.send_messages = None
+        submit_channel.set_permissions(ctx.guild.default_role, overwrite=overwrite)
+
+        return await info_channel.send(
+            f"<@&{CHALLENGE_PARTICIPANT_ROLE_ID}> Submissions are open." \
+            f"Upload your code file with extension in <#{CHALLENGE_SUBMIT_CHANNEL_ID}>. " \
+            f"Send `t.tag submitting` in <#{CHALLENGE_SUBMIT_CHANNEL_ID}> for more details",
+            allowed_mentions=discord.AllowedMentions(
+                roles=[
+                    discord.Object(CHALLENGE_PARTICIPANT_ROLE_ID)
+                ]
+            )
+        )
+    
+
+    @challenges_group.command(
+        name="close_submissions",
+        aliases=("cs",),
```

Comment:
You could add `close` as an alias. 
```suggestion
        aliases=("close", "cs"),
```